### ISSUE TITLE
Add additional Containerfile/Dockerfile based on alpine

### ISSUE THREAD

**Issue by emanuelb** [ez-org/eznode#11]
The Dockerfile used is based on debian, it would be helpful to have another one that is based on alpine.

**Comment by shesek** [comment]
I prefer not to maintain eznode on two different base images. I could consider switching entirely to alpine, but since the image is pretty large anyway (~100MB on top of the base image and expected to grow) I'm not sure there's much advantage to doing so.

Following our FB discussion on this matter, do you still see a need for this?

**Comment by emanuelb** [comment]
yes, all was explained in the FB discussion (the pros: are not just less space.., there many differences between alpine & debian) anyway it's something that probably worth to consider later after most of the external components already installed (to support lightning, other features , etc...)
